vpr/src/route/static_heap.cpp

```diff
@@ -0,0 +1,117 @@
+#include "static_heap.h"
+#include "binary_heap.h"
+#include "globals.h"
+
```

Comment:
Explain this file makes a single static heap that can be called from anywhere without an object pointer. Useful for 
backward compatibility with existing vpr code.

Reply:
Done